### PR TITLE
Adds content tools permissions and groups.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @washort (for migrations) @ngokevin

### CODE REVIEW

File: mkt/account/serializers.py [hunk 1]
```diff
@@ -129,6 +129,11 @@ def get_permissions(self, obj):
             'website_submitter': allowed('Websites', 'Submit'),
             'stats': allowed('Stats', 'View'),
             'revenue_stats': allowed('RevenueStats', 'View'),
+            'content_tools': {
+                'login': allowed('ContentTools', 'Login'),
+                'addon_submit': allowed('ContentTools', 'AddonSubmit'),
```
Comment (ngokevin) [inline mkt/account/serializers.py]: say we wanted to open this up to foxfooders, what would this look like? would we have to add each person to the group?

Comment (chuckharmston) [inline mkt/account/serializers.py]: Yup.

**Comment by washort** [comment]
r+

**Comment by chuckharmston** [comment]
@washort the migration did not appear to run when deployed to dev. Am I missing something?

	>>> from mkt.access.models import Group
	>>> Group.objects.all()
	[<Group: Admins>, <Group: Staff>, <Group: Statistic Viewers>, <Group: App Reviewers>, <Group: Senior Localizers>, <Group: Developers Credits>, <Group: Past Developers Credits>, <Group: Other Contributors Credits>, <Group: QA>, <Group: Revenue Stats Viewers>, <Group: System Administrators>, <Group: App Reviewer MOTD>, <Group: Developers>, <Group: Support Staff>, <Group: Restricted Users>, <Group: Senior App Reviewers>, <Group: Feature Managers>, <Group: Engagement>, <Group: No Reviewer Incentives>, <Group: Payment transactions clients>, '...(remaining elements truncated)...']
	>>> Group.objects.filter(name__startswith='Content Tools')
	[]

**Comment by washort** [comment]
odd, did it run locally?

**Comment by chuckharmston** [comment]
TASK: [migrate db] ************************************************************ 
	changed: [ip-172-31-12-48.ec2.internal] => {"changed": true, "cmd": "/data/zamboni/venv/bin/python manage.py migrate --noinput", "delta": "0:00:10.645075", "end": "2015-09-11 00:46:27.340430", "rc": 0, "start": "2015-09-11 00:46:16.695355", "stderr": "", "stdout": "Operations to perform:
	  Synchronize unmigrated apps: translations, djcelery, waffle, djangosecure, django_extensions, lookup, ecosystem, django, csp
	  Apply all migrations: feed, monolith, reviewers, inapp, site, api, es, ratings, webapps, tags, submit, access, contenttypes, developers, files, commonplace, users, sessions, zadmin, operators, auth, abuse, comm, webpay, prices, purchase, langpacks, admin, versions, websites, extensions
	Synchronizing apps without migrations:
	  Creating tables...
	  Installing custom SQL...
	  Installing indexes...
	Running migrations:
	  No migrations to apply.
	  Your models have changes that are not yet reflected in a migration, and so won't be applied.
	  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.", "warnings": []}

It looks like the `access` migration ran, which I guess means that `create_addon_groups` isn't being run.

**Comment by diox** [comment]
`mkt/account/models.py` is missing so the app is never considered to be a real app by django. Just adding an empty one seems to solve the issue locally.

**Pull request merged**